buf/registry/module/v1beta1/commit_service.proto

```diff
@@ -310,14 +310,28 @@ message GetBlobsResponse {
 }
 
 message GetMissingBlobDigestsRequest {
-  // The digests to see if we have Blobs for.
-  repeated buf.registry.storage.v1beta1.Digest digests = 1 [
+  message ModuleBlobDigests {
+    // The reference to the Module that the digests belong to.
+    ModuleRef module = 1;
```

Comment:
Should be `module_ref`, matching the rest of the API surface area.

When editing the APIs, be careful to make sure you are being consistent at all times.

Reply:
Fixed